src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -407,8 +422,23 @@ MavlinkReceiver::handle_message_command_int(mavlink_message_t *msg)
 	vcmd.param2 = cmd_mavlink.param2;
 	vcmd.param3 = cmd_mavlink.param3;
 	vcmd.param4 = cmd_mavlink.param4;
-	vcmd.param5 = ((double)cmd_mavlink.x) / 1e7;
-	vcmd.param6 = ((double)cmd_mavlink.y) / 1e7;
+
+	if (cmd_mavlink.x == INT32_MAX && cmd_mavlink.y == INT32_MAX) {
+		// INT32_MAX for x and y likely means to ignore it.
```

Comment:
Changing values based on speculation - not sure if I would do that.

Reply:
True, but it's in the spec now: https://github.com/mavlink/mavlink/pull/1634.
I'll change the comment.